src/js/charts/BarChart.js

```diff
@@ -15,6 +15,11 @@ class BarChart {
   constructor( { el, description, data, metadata } ) {
     data = processYoyData( data[0], metadata );
```

Comment:
Unrelated to this PR, if `data` is null or malformed we need to handle it gracefully.